porfolio.html

```diff
@@ -16,7 +16,41 @@
             <p class="text">John Doe</p>
```

Comment:
- For the `<div class="toolbar">` I suggest you change this  to a `<header>` element. Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

- For the `<div class="normal-button">` Kindly use a `<nav>` tag. The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.

- For the logo `John Doe` I suggest you use a link `<a>` tag  instead  of a `<p>`for this logo. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.